cuegui/cuegui/images/crystal/icons_rcc.py

```diff
@@ -23,7 +23,7 @@
 
 from PySide2 import QtCore
 
-qt_resource_data = "\
+qt_resource_data = b"\
```

Comment:
Note: Since these are compiled files, we will have to remember to compile against python 3 going forward.

Reply:
Ack - is there some doc to update on this? Perhaps we should just switch to use the icon image files directly instead of via this rcc file.